schemas/context/experienceevent.schema.json

```diff
@@ -26,9 +26,10 @@
         },
         "xdm:timestamp": {
           "title": "Timestamp",
-          "type": "string",
-          "format": "date-time",
-          "description": "The time when the first event of the interaction occurred."
+          "type": "integer",
```

Comment:
As pointed out in #117, I think this change is bad idea in multiple ways:

1. it introduces an inconsistency within XDM, because we now have some places where time is indicated through `date-time` and some where it is an `integer`
2. it makes XDM inconsistent with the rest of the JSON ecosystem that is using `string`-encoded dates all over the place.

The only reason I can see why we would do that is because it is more convenient for one single implementation, but that does not justify the price of this inconsistency for everyone else.